sources/api/host-containers/src/main.rs

```diff
@@ -229,14 +239,16 @@ where
         .output()
         .context(error::ExecutionFailure { command })?;
 
-    trace!("stdout: {}", String::from_utf8_lossy(&output.stdout));
+    let stdout = String::from_utf8_lossy(&output.stdout).to_string();
+
+    trace!("stdout: {}", stdout);
     trace!("stderr: {}", String::from_utf8_lossy(&output.stderr));
 
     ensure!(
         output.status.success(),
         error::CommandFailure { bin_path, output }
     );
-    Ok(())
+    Ok(stdout)
```

Comment:
Is there a near-future use case where stdout is handled by callers? If not, then I'd rather this stay as a unit `()` return. The `trace!` can keep printing the value, but I'm not sure it ought be returned to callers based on the changes at call sites in this PR.

Reply:
I used the `stdout` a little down as part of this change. I use the output to check what's the current default target, and force enable the host containers.

Reply:
Thanks, I missed that usage!